utils/auction.ts

```diff
@@ -251,3 +253,37 @@ export const verifyMessageWithTime = (data: MessageWithTimeSignature, signature:
   const addr = ethers.utils.verifyTypedData(domain, messageWithTimeType, data, signature!)
   return address.toLowerCase() === addr.toLowerCase()
 }
+
+export const validateOrder = async (order: Order, auction: Auction) => {
+  let isValidOrder = true;
+  let response = '';
+
+  if (order.isBuying != auction.isSelling) {
```

Comment:
does this work ok if the auction details haven't been finalized? (same for min size below)

Reply:
By finalized u mean auction not started yet/auction details not in the database?

Reply:
Users can submit bids even before the minSize, auction direction is set. If it's not a valid bid manager will omit these bids. 

During this minSize will be 0. So it should by pass this check if the auction is not yet started